_benchmark/reference/workloads/operations.md

```diff
@@ -0,0 +1,323 @@
+
+### Meta-data
+
+The `create-index` operation returns the following meta-data:
+
+`weight`: The number of indexes created by the operation.
+`unit`: Always “ops”.
+`success`: A Boolean indicating whether the operation has succeeded.
+
+## delete-index
+
+The `delete-index` runs the [Delete Index API](api-reference/index-apis/delete-index/). Like the [`create-index`](#create-index) operation, you can delete all indexes found in the `indices` section of the workload or you can delete one or more indexes based on string passed in the `index` setting.
+
+### Usage
+
+The following example deletes all indexes found in `indices` section of the workload:
+
+```yml
+{
+  "name": "delete-all-indices",
+  "operation-type": "delete-index"
+}
+```
+
+The following example deletes all `logs_*` indexes:
+
+```yml
+{
+  "name": "delete-logs",
+  "operation-type": "delete-index",
+  "index": "logs-*",
+  "only-if-exists": false,
+  "request-params": {
+    "expand_wildcards": "all",
+    "allow_no_indices": "true",
+    "ignore_unavailable": "true"
+  }
+}
+```
+
+### Options
+
+Use the following options when deleting all indexes indicated in the `indices` section of the workload.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`only-if-exists` | No | Boolean | Decides whether an index should only be deleted in the index exists. Default is `true`.
+`request-params` | No | List of settings | Contains any request parameters allowed by the Create Index API. OpenSearch Benchmark will not attempt to serialize the parameters and pass them as is. 
+
+Use the following options if you want to delete one or more indexes based on pattern indicated in the `index` option:
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`index` | Yes | String | The name of the index or indexes you want to delete. 
+`only-if-exists` | No | Boolean | Decides whether an index should only be deleted in the index exists. Default is `true`.
+`request-params` | No | List of settings | Contains any request parameters allowed by Create Index API. OpenSearch Benchmark will not attempt to serialize the parameters and pass them as is. 
+
+### Meta-data
+
+The `delete-index` operation returns the following meta-data.
+
+`weight`: The number of indexes created by the operation.
+`unit`: Always “ops”.
+`success`: A Boolean indicating whether the operation has succeeded.
+
+## cluster-health
+
+The `cluster-health` operation runs the [Cluster Health API](api-reference/cluster-api/cluster-health/), which checks the cluster health status returns the expected status according the parameters set in the `request-params` option. If an unexpected cluster health status is returned, the operation reports a failure. You can use the `--on-error` option in the OpenSearch Benchmark `execute-test` command to control how OpenSearch Benchmark behaves when an the health check fails.
```

Comment:
`checks the cluster health status returns the expected` -> `checks the cluster health status and returns the expected`